config/swagger.yaml

```diff
@@ -2245,7 +2247,7 @@ definitions:
     type: object
     properties:
       oracle_pubkey:
-        type: string
+        $ref: '#/definitions/EncodedHashOrName'
```

Comment:
I see [serialization documented as binary without size](https://github.com/aeternity/protocol/blob/0918c2a3262bb6e39e9eb589da380b248d006f3a/serializations.md#oracle-query-transaction). So no need to change that. Ok.

Reply:
should clarify on what the contents is allowed to be